_scripts/reconfigure_oauth2

```diff
@@ -35,9 +39,28 @@ if [[ ${1} =~ ^default= ]]; then
     if [[ ${1} =~ ^default=yes$ ]] || [[ $(${BIN}/dotenv -f ${ENV_FILE} get ${VAR}) == "yes" ]]; then
         DEFAULT_ENABLED_OAUTH2=yes
     fi
-    echo ""
-    echo "Using OpenID/OAuth2 will require a login to access your app, but it will not affect what a successfully logged-in person can do in your app. If your app has a built-in authorization mechanism that can check for the user header that traefik-forward-auth sends, then your app can limit what the logged-in person can do in the app. But if your app can't check the user header, or if your app doesn't have built-in authorization at all, then any person with an account on your Gitea server can log into your app and have full acces"
-    #$(${BIN}/confirm $$([[ $$(${BIN}/dotenv -f ${ENV_FILE} get ${VAR}) == "true" ]] && echo "yes" || echo "no") "Do you want to enable OpenID/OAuth2" "?" && ${BIN}/reconfigure ${ENV_FILE} ${VAR}=true || ${BIN}/reconfigure ${ENV_FILE} ${VAR}=false)
-    ${BIN}/confirm ${DEFAULT_ENABLED_OAUTH2} "Do you want to enable OpenID/OAuth2" "?" || disable_oauth2
+    echo 
+    echo "Using OpenID/OAuth2 will require a login to access your app. You can configure basic authorization by assigning an authorization group (authorization groupscontain email addresses that are allowed to log into your app - you configure authorization groups by running \`make groups\` in the \`traefik\` directory)."
+    echo 
+    echo "SECURITY NOTE: Using OpenID/OAuth2 is on top of any authentication/authorization service your app provides. OpenID/Oauth2 will require a login to access your app and permit only specific logins, but it will not affect what a successfully logged-in person can do in your app. If your app has a built-in authorization mechanism that can check for the user header that traefik-forward-auth sends, then your app can limit what the logged-in person can do in the app. But if your app can't check the user header, or if your app doesn't have built-in authorization at all, then any person with an account on your Gitea server that you permit to log into your app will have full access."
```

Comment:
I think the security note is out of date assuming that we enforcing the groups, its not a concern anymore.

Reply:
You want me to remove the entire paragraph? from `reconfigure_oauth2` (i.e., the prompt when you're starting to configure oauth2), from the app readmes, and from the t-f-a readme?

Reply:
how about replacing it with:

OAuth2 uses traefik-forward-auth to delegate authentication to an external authority (eg. a self-deployed gitea instance). Accessing this app will require all users to login through that external service first. Once authenticated, you may be authorized access only if your login id matches the member list of the predefined security group configured for the app (`WHOAMI_OAUTH2_AUTHORIZED_GROUP`) . Security groups are defined in the Traefik config (`TRAEFIK_HEADER_AUTHORIZATION_GROUPS`) and can be modified using `make groups` (in the traefik directory).

Reply:
> You want me to remove the entire paragraph? from `reconfigure_oauth2` (i.e., the prompt when you're starting to configure oauth2), from the app readmes, and from the t-f-a readme?

I think its probably good to have it in each app readme, but specifically we dont need to worry about telling the user that "any user can do whatever they want" because now they have to be a part of a group.

Reply:
updated.